meilisearch_fastapi/models/search_parameters.py

```diff
@@ -19,3 +19,6 @@ class SearchParameters(CamelBase):
     highlight_pre_tag: str = "<em>"
     highlight_post_tag: str = "</em>"
     crop_marker: str = "..."
+    matching_strategy: str = "all"
+    hists_per_page: int | None = None
+    page: int | None = None
```

Comment:
You did this correctly according to the issue, but I am realizing looking at it now that Pydantic is not going to like this in Python 3.8 so can you make the following update?

```suggestion
    hits_per_page: Optional[int] = None
    page: Optional[int] = None
```